scss/_grid.scss

```diff
@@ -10,7 +10,7 @@ Mobile First - This applies from 0px to 767px
 
 /***** Row setup *****/
 .row                                            { margin:0 auto; padding:0 20px; display:flex; flex-direction:row; box-sizing:border-box; flex:0 1 auto; flex-wrap:wrap; }
-.row.row--width-fluid                           { width:100%; }
+.row .row--width-fluid                          { width:100%; }
```

Comment:
I had added .row.row--width-fluid as the class will get rewritten later by the standard .row and didn't want to have to repeat myself for each media query.